carma_cooperative_perception/config/cp_sdsm_to_detection_list_node.yaml

```diff
@@ -0,0 +1,28 @@
+# Whether to overwrite the default covariance values with those specified below
+overwrite_covariance: true
+
+##################################################
+# NONE of the covariance values can be set to zero!
+##################################################
+
+# Pose position covariance values (in meters^2)
+# These represent uncertainty in position measurements
+pose_covariance_x: 0.02  # X-axis position uncertainty (lateral)
+pose_covariance_y: 0.02  # Y-axis position uncertainty (longitudinal)
+pose_covariance_z: 0.02  # Z-axis position uncertainty (vertical)
+
+# Pose orientation covariance value (in radians^2)
+# 10% error in yaw angle, which is (0.1 * 2 pi) ^ 2 = 0.394384
+pose_covariance_yaw: 0.394384  # Yaw angle uncertainty (heading)
+
+# Twist linear velocity covariance values (in (m/s)^2)
+# These represent uncertainty in velocity measurements
+twist_covariance_x: 0.005  # X-axis velocity uncertainty
+twist_covariance_z: 0.005  # Z-axis velocity uncertainty
+
+# Twist angular velocity covariance value (in (rad/s)^2)
+twist_covariance_yaw: 0.005  # Yaw angular velocity uncertainty
+
+adjust_position: true
```

Comment:
If the sensor calibration is good, we wouldn't need this. However, since sensor calibration takes time, this option gives us more last minute tuning of the sensor data.

Reply:
Added the comment as well as yaw offset